address-space-controller/src/main/java/io/enmasse/controller/CreateController.java

```diff
@@ -273,4 +284,29 @@ private InfraConfig getInfraConfig(AddressSpace addressSpace) {
     public String toString() {
         return "CreateController";
     }
+
+    private Set<HasMetadata> getExistingResources(String infraUuid) {
+        Set<HasMetadata> existingResources = Collections.emptySet();
+        if (infraUuid != null) {
+            existingResources = kubernetes.getResources(infraUuid).stream().filter(m -> m.getMetadata().getAnnotations() == null || !m.getMetadata().getAnnotations().containsKey(AnnotationKeys.CLUSTER_ID)).collect(Collectors.toSet());
```

Comment:
Wouldn't this accidentally delete resources that are not created in the createInfraResources? I.e. RouterConfigController, NetworkPolicyController, AuthController etc. all create resources with infra uuid.

Reply:
Yeah, you are right.  I failed to consider those.   I was fixating on the broker resources which are identifiable by their annotation.   I don't see a clean way to identify only those the were created from the infra template in order to handle this generally.

Reply:
Ok, so maybe for now, we just identify which resources was created that are no longer needed. Is it the client secret for the console that is removed?

Reply:
No, the oauth client secret remains required.    The resources I need to delete are the console-xxxxxxx service and the console-xxxxxx route.